paasta_tools/setup_prometheus_adapter_config.py

```diff
@@ -69,7 +69,7 @@
 DEFAULT_EXTRAPOLATION_PERIODS = 10
 DEFAULT_EXTRAPOLATION_TIME = DEFAULT_SCRAPE_PERIOD_S * DEFAULT_EXTRAPOLATION_PERIODS
 
-CPU_METRICS_PROVIDERS = ("mesos_cpu", "cpu")
+CPU_METRICS_PROVIDERS = "cpu"
```

Comment:
this should remain a collection or the code that uses this should be refactored to use equality (rather than `in`)

Reply:
(i'd probably switch to equality and either drop this constant or un-plural the constant name)